src/pudl/package_data/eia860m/column_maps/generator_existing.csv

```diff
@@ -15,7 +15,7 @@ winter_capacity_mw,,,,,,,,,,,,,net_winter_capacity_mw,net_winter_capacity_mw,net
-sector_name,sector_name,sector_name,sector_name,sector_name,sector_name,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector
+sector_name_eia,sector_name,sector_name,sector_name,sector_name,sector_name,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector
```

Comment:
Switched to using the canonical name that we've defined elsewhere.

Reply:
good good! This also needs to be updated in the other two `generator_*` column maps/

Reply:
Ooh, whooops! Good catch.

Reply:
Maybe this will improve coverage actually.